src/NuGetPackageVerifier/PackageIssueFactory.cs

```diff
@@ -109,6 +120,12 @@ private static PackageVerifierIssue AssemblyMissingVersionAttributeCore(string i
             return new PackageVerifierIssue(issueId, assemblyPath, string.Format("The managed assembly '{0}' in this package is missing the '{1}' attribute.", assemblyPath, attributeName), PackageIssueLevel.Error);
         }
 
+        public static PackageVerifierIssue AuthorWrong(string assemblyPath, string expectedAuthor)
+        {
+            return new PackageVerifierIssue("PACKAGE_AUTHOR_WRONG", assemblyPath, string.Format(
+                "The managed assembly '{0}'s Author must be {1}", assemblyPath, expectedAuthor), PackageIssueLevel.Error);
```

Comment:
This is ultimately about the *package* having a wrong/missing author, not the assembly.